sass/util/_width.scss

```diff
@@ -0,0 +1 @@
+.mw8 { max-width: 20rem; }
```

Comment:
Good catch, I think that's a holdover from a previous attempt. It should definitely be `mw20`. Will change